src/models/multiChart.js

```diff
@@ -213,6 +213,9 @@ nv.models.multiChart = function() {
                 return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
             }).concat([{x:0, y:0}]) : [];
 
+            x = lines1.xScale();
```

Comment:
X was initialized to a linear scale and that scale was applied to the xAxis. However changing it afterwards didn't work. 

So the initialization was removed and x is only assigned here and used afterwards. 

It is possible I missed something, but this was the only way I got a different xAxis scale working.